runhouse/resources/hardware/cluster_factory.py

```diff
@@ -307,6 +307,10 @@ def ondemand_cluster(
         else open_ports
     )
 
+    # Add the new Runhouse Ray port to open_ports if it's not already there
+    if Cluster.DEFAULT_RAY_PORT not in open_ports:
+        open_ports.append(Cluster.DEFAULT_RAY_PORT)
+
```

Comment:
😬 makes me nervous. If we use internal IPs do we really need to open it? Wouldn't this expose the cluster to the outside world?

Reply:
Works without, removed.